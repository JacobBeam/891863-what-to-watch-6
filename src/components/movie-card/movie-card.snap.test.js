import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import MovieCard from './movie-card';

const film = {id: 0, name: `test`}
const onAddActiveCard = jest.fn();
const onDeleteActiveCard = jest.fn();
it(`Should MovieCard render correctly`, () => {
  const history = createMemoryHistory();

  const { container } = render(
    <Router history={history}>
      <MovieCard
        film={film}
        onAddActiveCard={onAddActiveCard}
        onDeleteActiveCard={onDeleteActiveCard} />
    </Router>);
  expect(container).toMatchSnapshot();
});
