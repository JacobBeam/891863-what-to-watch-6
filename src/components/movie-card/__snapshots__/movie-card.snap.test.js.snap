// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieCard render correctly 1`] = `
<div>
  <article
    class="small-movie-card catalog__movies-card"
    data-film-id="0"
  >
    <video
      data-testid="videoplayer"
      height="175"
      width="280"
    />
    <h3
      class="small-movie-card__title"
    >
      <a
        class="small-movie-card__link"
        href="/films/0"
      >
        test
      </a>
    </h3>
  </article>
</div>
`;
